engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -2292,6 +2293,38 @@ uint16 Myst::rocketSliderGetSound(uint16 pos) {
 	return (uint16)(9530 + (pos - 216) * 35.0 / 61.0);
 }
 
+
+uint16 Myst::rocketCheckIfSoundMatches(uint16 sound1, uint16 sound2) {
+
+	bool fuzzylogic = false;
+
+  	if (ConfMan.getBool("fuzzy_logic")) {
+		fuzzylogic = true;
+	}
+
+	debugN("helper: %i %i %i (diff:% 3i) ", sound1, sound2, fuzzylogic, sound1 - sound2);
+
+	if (fuzzylogic == false) {
+
+		if (sound1 != sound2) {
+			debugN("strict : false \n");
+			return false;
+		}
+
+	} else {
+
+	       if (abs(sound1 - sound2) >= 5) {
+			debugN(" fuzzy : false \n");
+			return false;
+		}
```

Comment:
You can remove the if and change this to:

    return abs(sound1 - sound2) < 5;